src/pkg/packager/common.go

```diff
@@ -219,6 +222,13 @@ func (p *Packager) loadZarfPkg() error {
 		return fmt.Errorf("unable to read the zarf.yaml in %s: %w", p.tmp.Base, err)
 	}
 
+	// Validate the checksums of all the things!!!
+	if p.cfg.Pkg.Metadata.ChecksumSignature != "" {
```

Comment:
Can we call this something else since technically it isn't a signature (just a checksum of checksums):  `Metachecksum`, `AggregateChecksum` or something similar might make sense